### PR TITLE
data table scrolling

### PR DESCRIPTION
**Pull request by awuBullhorn**
## **Description**

INSERT A SMALL DESCRIPTION OF WHAT YOU CHANGED HERE

#### **Verify that...**

- [x] Any related demos were added and `npm start` and `npm run build` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [x] `npm test` passes and code coverage is increased
- [x] `npm run build` still works

#### **Bullhorn Internal Developers**
- [x] Run `Novo Automation`
- [x] Run `BBO Automation`

##### **Screenshots**

### CODE REVIEW

File: projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css [hunk 1]
```diff
@@ -1 +1,8 @@
 /** No CSS for this example */
+.fixedWindowSize {
+  height: 600px;
+}
+
```
Comment (amrutha-rajiv) [inline projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css]: the height is always going to be 600px?

Comment (awuBullhorn) [reply projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css]: It's just for this demo example to create a scrollable window so you can test/see the sticky header in action.

Comment (amrutha-rajiv) [reply projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css]: aha. sorry, Missed the example piece :)

File: projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css [hunk 2]
```diff
@@ -1 +1,8 @@
...
+novo-data-table {
+  height: 100%;
```
Comment (charlesabarnes) [inline projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css]: This part is important for the fixed header feature.  LGTM

**Pull request merged**